components/k8s-model-server/http-proxy/server.py

```diff
@@ -129,7 +188,9 @@ def get_application(**settings):
     return tornado.web.Application(
             [
             (r"/model/(.*):predict", PredictHandler),
+            (r"/model/(.*):classify", ClassifyHandler),
```

Comment:
Can you add docs explaining the difference between predict and classify?

Reply:
so with classify method, the default model we have `gs://kubeflow-models/inception`  would finally work with http-proxy.

Reply:
So the difference is ClassifyHandler expects as input a json dictionary with fields model_spec and input where as PredictHandler expects a map of names to tensor?

Can you update README.md to explain this?